rest-api/api/base_api.py

```diff
@@ -44,6 +63,42 @@ def _get_request_arg_bool(self, key, default=False):
       return default
     return request.args.get(key).lower() == 'true'
 
+  def _save_raw_request(self, obj):
+    """ Create deferred task to save the request payload and possibly link it to a table record """
+    log = RequestsLog()
+
+    log.endpoint = request.endpoint
+    log.method = request.method
+    log.url = request.url
+    log.resource = request.data
+    log.version = int(request.url.split('/')[4][1:])
+
```

Comment:
I'm a little confuse about this `log.version`,  do we always have this version value in the url? Do we need to catch the exception here?

Reply:
As long as we use following path format `https://xxxx/rdr/v1/xxxx` , it will return 1 or what ever integer is the version number.

Reply:
Oh, I forgot the prefix. Thanks for the explanation.